src/OpenSage.Game/Logic/Object/Draw/W3dModelDraw.cs

```diff
@@ -425,6 +425,8 @@ internal sealed class W3dModelDrawConditionState : DisposableBase
 
             _particleSystems = particleSystems;
             _context = context;
+
+            AddDisposeAction(() => Deactivate());
```

Comment:
Super nitty nitpick that doesn't matter but I'll point out anyway ;)
```suggestion
            AddDisposeAction(Deactivate);
```

Reply:
yeah, looks a bit easier to read